src/Tag/DiscontinuitySequenceTag.php

```diff
@@ -33,11 +33,14 @@ public function getDiscontinuitySequence()
 
     public function dump()
     {
+        if (empty($this->discontinuitySequence) && $this->discontinuitySequence !== '0') {
```

Comment:
I think `null === $this->discontinuitySequence` is enough. if someone don't need `discontinuitySequence`, he/she should not call the setter.